proto/rill/admin/v1/api.proto

```diff
@@ -494,6 +520,54 @@ message SudoGetResourceResponse {
   }
 }
 
+// SudoGetOrganizationQuotaRequest
+message SudoGetOrganizationQuotaRequest {
+  string org_name = 1;
+}
+
+// SudoGetOrganizationQuotaResponse
+message SudoGetOrganizationQuotaResponse {
+  OrganizationQuota organization_quota = 1;
+}
+
+// SudoGetUserQuotaRequest
+message SudoGetUserQuotaRequest {
+  string email = 1;
+}
+
+// SudoGetUserQuotaResponse
+message SudoGetUserQuotaResponse {
+  UserQuota user_quota = 1;
+}
+
+// SudoSetOrganizationQuotaRequest
+message SudoSetOrganizationQuotaRequest {
+  string org_name = 1;
+  oneof quota {
+    uint32 quota_projects = 2;
+    uint32 quota_deployments = 3;
+    uint32 quota_slots_total = 4;
+    uint32 quota_slots_per_deployment = 5;
+    uint32 quota_outstanding_invites = 6;
+  }
```

Comment:
Allow setting more than one quota, using something like:
```
string org_name = 1;
optional uint32 quota_projects = 2;
optional uint32 ...
```

See the proto changes and implementation in this PR for inspiration: https://github.com/rilldata/rill/pull/2582